src/ethereum_clis/clis/eels_t8n.py

```diff
@@ -1,139 +0,0 @@
-"""
-EELS T8N transition tool.
-
-https://github.com/ethereum/execution-specs
-"""
-
-from abc import ABC, abstractmethod
-from typing import List
-
-from ethereum_clis import TransitionTool, TransitionToolOutput
-from ethereum_test_base_types import BlobSchedule
-from ethereum_test_forks import Fork
-from ethereum_test_types import Alloc, Environment, Transaction
-
-
-class EELST8NWrapper(TransitionTool, ABC):
```

Comment:
I wanted to have certain constraints/guarantees on the implementation of the EELS t8n wrapper within the EEST framework. Hence I added the abstract base class. Are we ok with removing those?